app/views/shared/feedback_forms/_form.html.erb

```diff
@@ -35,6 +35,14 @@
           <%= email_field_tag :to, "", class:"form-control" %>
         </div>
       </div>
+      <% unless current_user? %>
+        <div class="form-group row library-hours-captcha">
+          <div class="col-md-9 offset-md-3">
+            <%= recaptcha_tags %>
+            <p>(Stanford users can avoid this Captcha by logging in.)</p>
```

Comment:
Maybe this message should go in the translation file?

Reply:
that makes sense to me. iirc this is just copied verbatim from searchworks. idk if i'm confident enough with rails i18n to know how to do that in a template (is it just the `t()` function?)

Reply:
Yeah, something like: `<%= t('library-hours.feedback_form.log_in_to_avoid_captcha') %>`